### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by zimbatm**
Optimistic lock on #370

Based on my knowledge of the people involved, it might take a while, but I don't think anybody will be against licensing under MIT.

Meanwhile, this allows the clarification of new contributions under the license.

### CODE REVIEW

**Comment by delroth** [comment]
> Meanwhile, this allows the clarification of new contributions under the license.

Unfortunately it also implies that the contents of the repo are MIT licensed, which they currently aren't (at least not completely). So NAK on merging this before we have collected approvals.

I get your intention though - but I don't think it's necessary. People with direct access to the repo have all already approved or are already in the list, and in the eventuality of a PR from a new contributor we can ask them in the PR.

**Comment by zimbatm** [comment]
Everybody approved now

**Review by JulienMalka — APPROVED** [review]

**Pull request merged**
